assets/queries/cloudFormation/aws/s3_bucket_allows_get_actions_from_all_principals/query.rego

```diff
@@ -7,6 +7,8 @@ CxPolicy[result] {
 	resource := input.document[i].Resources[name]
 	resource.Type == "AWS::S3::BucketPolicy"
 
+	cf_lib.has_bucket_associated(input.document[i].Resources, resource)
```

Comment:
```suggestion
````

The same applies for the other queries that only look for AWS::S3::BucketPolicy.